utils/content.js

```diff
@@ -0,0 +1,8 @@
+export function isContentTooShort(content, minimumNumWords) {
+  const regexPattern = new RegExp(`[a-záéíóúâôãõç]{${minimumNumWords},}`, 'gi');
+  const matches = content.normalize('NFC').match(regexPattern);
+
+  if (!matches) return true;
+
+  return matches.length < minimumNumWords;
```

Comment:
Aqui pode simplificar:

```suggestion
  return !matches || matches.length < 5;
```